content/threads_intro.tex

```diff
@@ -0,0 +1,73 @@
+This section specifies the interaction between \openshmem{} interfaces and the
+user threads, and also describes the routines that can be used for initializing and 
+querying the thread environment.
+ 
+
+\begin{itemize}
+
+\item
+In a thread-compliant implementation, an \openshmem{} \ac{PE} is multithreaded. 
+Each thread can issue \openshmem{} calls; however the threads
```

Comment:
I'm having trouble reconciling the two statements:
* In a thread-compliant implementation, an OpenSHMEM is multithreaded. 
* Each thread can issue OpenSHMEM calls; ...

Is the latter point part of the definition of a "thread-compliant implementation"? If so, then an implementation that supports SHMEM_THREAD_FUNNELED is not "thread compliant".